hetmech/hetmat.py

```diff
@@ -442,12 +443,24 @@ def set(self, metapath, metric, damping, matrix, runtime):
         """
         pass
 
+    def get_stats(self):
+        """
+        Return a str with formatted stats about cache operations
+        """
+        hits_str = ', '.join(f'{kind} = {count:,}' for kind, count in self.hits.items())
+        stats_str = textwrap.dedent(f'''\
+            {self.__class__.__name__} containing {len(self.cache):,} items
+              total gets: {sum(self.hits.values()):,}
+              cache hits: {hits_str}''')
+        return stats_str
+
 
 class PathCountPriorityCache(PathCountCache):
 
     def __init__(self, hetmat, allocate_GB):
         super().__init__(hetmat)
-        self.allocate_B = 1_000_000_000 * allocate_GB
+        self.bytes_per_gigabyte = 1_000_000_000
```

Comment:
Why do we want this as an attribute, when `self.bytes_per_gigabyte` is longer than the number itself or `1e9`, which are equivalent?

Reply:
1e9 creates float. I felt that there is less of a change to get an error if we only set the conversion ratio once and it's semantically more expressive.